scraper.ts

```diff
@@ -163,32 +149,18 @@ function parseData(content: string, errorURIs: string[], trie: Trie) {
   return uniq(filtered);
 }
 
-function parseAnchorSection(section: string) {
-  const [
-    term,
-    type,
-    spec,
-    shortname,
-    level,
-    status,
-    uri,
-    isExported,
-    normative,
-    ...forContext
-  ] = section.split('\n');
-
-  const dataFor = forContext.filter(Boolean);
-  const normalizedType = CSS_TYPES_INPUT.has(type) ? `css-${type}` : type;
+function mapDefinition(dfn: WebrefDfn, term: string, spec: string, series: string) {
+  const normalizedType = CSS_TYPES_INPUT.has(dfn.type) ? `css-${dfn.type}` : dfn.type;
   return {
     term: normalizeTerm(term, normalizedType),
-    isExported: isExported === '1',
+    isExported: dfn.access === 'public',
     type: normalizedType,
     spec,
-    shortname,
-    status,
-    uri, // This is full URL to term here
-    normative: normative === '1',
-    for: dataFor.length > 0 ? dataFor : undefined,
+    shortname: series,
+    status: "current",
+    uri: '#' + dfn.id, // This is full URL to term here
```

Comment:
This is kinda breaking change, for HTML specs, as it no longer has the path.

```diff
-   "uri": "iframe-embed-object.html#attr-object-type",
+   "uri": "#attr-object-type",
```

Reply:
Instead of `dfn.id`, we should be using `dfn.href.replace(spec.url)`. I kinda of want to avoid the multipage redirect.

https://html.spec.whatwg.org/multipage/#attr-object-type -> https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-object-type